### DISCUSSION TITLE
Add inertia-js

### DISCUSSION THREAD

**Discussion by pascal-codetaal** [adonisjs/core#1300] [Help]
I would like to add  [adonis-inertia](https://github.com/RomainLanz/adonis-inertia
) to my adonis V5 project

I first installed adonis inertia via
`yarn add adonis-inertia`

After that i invoked following command
`node ace invoke adonis-inertia`

But now i'm stuck, @RomainLanz could point me in the right direction.

I could write a small how to add tutorial after that.

**Comment by woleEkanola** [comment]
@pascal-codetaal Hi did you make any progress with this?

**Reply by pascal-codetaal** [reply]
@woleEkanola nope i gave up :-(
